### PR TITLE
Onboarding

### PR DESCRIPTION
**Pull request by bdauphouy**

### CODE REVIEW

File: src/shared/utils/atom-with-listeners.ts
Comment (arthur-fontaine) [inline src/shared/utils/atom-with-listeners.ts:1]: This file can be removed.

File: src/translations/en.ts
Comment (arthur-fontaine) [inline src/translations/en.ts:1]: We might want to add a "." at the end of the onboarding descriptions.

File: src/translations/en.ts [hunk 1]
```diff
@@ -1,6 +1,28 @@
...
 export const enTranslations: Translations['en'] = {
+  onboarding: {
+    pages: {
+      0: {
+        title: 'Welcome to Karmine Corp',
+        description: 'Follow the matches of your favorite teams',
+      },
+      1: {
+        title: 'Notifications',
+        description: 'Receive notifications to not miss anything',
```
Comment (arthur-fontaine) [inline src/translations/en.ts:12]: Invite the user to select the game for which they want to receive notifications.

File: src/translations/en.ts [hunk 2]
```diff
@@ -1,6 +1,28 @@
...
+      },
+      2: {
+        title: 'Spoilers',
+        description: "Don't want to know the results? Turn off the spoilers",
```
Comment (arthur-fontaine) [inline src/translations/en.ts:16]: Explain to the user how to turn off spoilers.

**Review by arthur-fontaine — CHANGES_REQUESTED** [review]
French and Spanish translations are missing.

File: src/translations/en.ts [hunk 3]
```diff
@@ -1,6 +1,28 @@
 import { Translations } from './Translations';
 
 export const enTranslations: Translations['en'] = {
+  onboarding: {
+    pages: {
+      0: {
+        title: 'Welcome to Karmine Corp',
```
Comment (arthur-fontaine) [inline src/translations/en.ts:7]: It should be "Welcome to Karmine Board"

File: src/translations/en.ts [hunk 4]
```diff
@@ -1,6 +1,28 @@
...
+        description: 'Follow the matches of your favorite teams',
```
Comment (arthur-fontaine) [inline src/translations/en.ts:8]: It should be "Follow the matches of Karmine Corp and more"

**Review by arthur-fontaine — CHANGES_REQUESTED** [review]

File: src/screens/onboarding/modals/onboarding-modal.tsx [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+  }, []);
+
+  const { currentStep, totalSteps, title, description, handleNext, children } =
+    useOnboardingController({
+      handleEnd,
+    });
+
+  useEffect(() => {
+    console.log('Current step:', currentStep);
+  });
```
Comment (arthur-fontaine) [inline src/screens/onboarding/modals/onboarding-modal.tsx:34]: It should be removed.

File: src/screens/onboarding/children/step-2.tsx [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+import { SettingsContext } from '~/shared/contexts/settings-context';
+import { useStyles } from '~/shared/hooks/use-styles';
+import { createStylesheet } from '~/shared/styles/create-stylesheet';
+
+export const Step2 = () => {
+  const { settings, setSettings } = useContext(SettingsContext);
+
+  const styles = useStyles(getStyles);
+
+  const match = useMemo<CoreData.Match>(
```
Comment (arthur-fontaine) [inline src/screens/onboarding/children/step-2.tsx:17]: We might want to put KC vs. T1 and the competition name LEC.

File: src/shared/utils/atom-with-listeners.ts
Comment (bdauphouy) [reply src/shared/utils/atom-with-listeners.ts:1]: Why? We're still using it in the splash screen logic. Maybe another branch is still waiting to be merged to main and could contains the modifications you are thinking of.

**Review by arthur-fontaine — APPROVED** [review]

**Pull request merged**